doc/function_types.md

```diff
@@ -1036,12 +1036,12 @@ int main()
   // use the PrimalDualSolver to solve it.
   // ens::PrimalDualSolver could be replaced with ens::LRSDP or other ensmallen
   // SDP solvers.
-  PrimalDualSolver<SDP<arma::sp_mat>> solver(sdp);
+  PrimalDualSolver<SDP<arma::sp_mat>> solver;
```

Comment:
```suggestion
  PrimalDualSolver solver;
```

Correct me if I'm wrong on this. :+1:

Reply:
I agree with you @rcurtin.  I will add the history entry and update function_types.md and resubmit.